### PR TITLE
GHC-81: Add Programs

### PR DESCRIPTION
**Pull request by nuhuhmutebi**

### CODE REVIEW

File: configuration/concepts/program_attribute_types.csv [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+5dafe35b-94fb-4978-8f48-e92950f52c36,,,Gynecology,Gynecology,,,,,Misc,N/A,,,,,,,,,,,
+75bc6f17-d087-4c28-9f5a-da98ed3812d4,,,Ophthalmology,Ophthalmology,,,,,Misc,N/A,,,,,,,,,,,
+ee4ac5ca-7be1-44cb-a750-dba64720453a,,,Dental,Dental,,,,,Misc,N/A,,,,,,,,,,,
+1c64a247-fc10-4e12-982c-c73859ed3782,,,Family Medicine,Family Medicine,,,,,Misc,N/A,,,,,,,,,,,
+b2017b29-ee16-4c62-8d3e-5ff2c2d3ca07,,,Internal Medicine,Internal Medicine,,,,,Misc,N/A,,,,,,,,,,,
+bef52504-d77b-485c-ab81-9ccb6eb8f239,,,Pediatrics,Pediatrics,,,,,Misc,N/A,,,,,,,,,,,
+2484cdc7-a77a-4305-8bcc-b9c90f384bf6,,,Cardiology,Cardiology,,,,,Misc,N/A,,,,,,,,,,,
+dc04cfe1-29d1-46a7-a0cf-b8baacfdbb31,,,Dermatology,Dermatology,,,,,Misc,N/A,,,,,,,,,,,
+13aa8824-8a29-46c9-b80e-9201f03e82e5,,,Wound,Wound,,,,,Misc,N/A,,,,,,,,,,,
+eeed1828-3038-4997-ada8-55f75c0d33db,,,Other,Other,,,,,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/program_attribute_types.csv]: This concept already exists in the 'others.csv'. Let us not duplicate them

Comment (nuhuhmutebi) [reply configuration/concepts/program_attribute_types.csv]: Deleted

Comment (reagan-meant) [reply configuration/concepts/program_attribute_types.csv]: @nuhuhmutebi  Change the file order such that 'Other.csv' is loaded first before this file

Comment (nuhuhmutebi) [reply configuration/concepts/program_attribute_types.csv]: Done

File: configuration/concepts/program_attribute_types.csv [hunk 2]
```diff
@@ -0,0 +1,36 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Description:es,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
+c969a930-b6f3-4b73-b93e-52defe1c9217,,,1,1,,,,,Misc,N/A,,,,,,,,,,,
+e0c53584-3e26-4edd-94d8-0f46b3333c49,,,2,2,,,,,Misc,N/A,,,,,,,,,,,
+01d56851-89c5-487b-836a-2783f826b1e7,,,3,3,,,,,Misc,N/A,,,,,,,,,,,
+347c7d54-0a11-4120-8016-b2e596ec75b8,,,4,4,,,,,Misc,N/A,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/program_attribute_types.csv]: Let us use these [concepts](https://github.com/globalhealthcoalition/openmrs-config-ghc/blob/04d5bd875467fafcc8bc975aef8ebc950b4ecbf8/configuration/concepts/checkout.csv#L3) instead

Comment (nuhuhmutebi) [reply configuration/concepts/program_attribute_types.csv]: Done

**Pull request merged**